dataland-qa-service/src/main/kotlin/org/dataland/datalandqaservice/services/QaLogMessageBuilder.kt

```diff
@@ -0,0 +1,55 @@
+package org.dataland.datalandqaservice.org.dataland.datalandqaservice.services
+
+import org.springframework.stereotype.Component
+
+/**
+ * A builder for log messages related to QA reports.
+ */
+@Component
+class QaLogMessageBuilder {
```

Comment:
Jede dieser Nachrichten wird genau ein mal verwendet. Ist das einführen dieses Service nicht overkill?

Reply:
Grundsätzlich finde ich es persönlich gut, wenn wir thematisch verwandte log-messages in eigenen Klassen/Files zentral managen.
Das ist angenehmer zu lesen und wartbarer.
Wir machen das z.B. auch mit dem `LogMessageBuilder` service im backend, oder mit dem `DataRequestLogger` im community manager.

Können aber auch gerne nochmal darüber sprechen.